pkgs/top-level/all-packages.nix

```diff
@@ -33892,6 +33892,9 @@ with pkgs;
     toolsOnly = true;
   };
 
+  # variant of qemu building user space emulator only - intended to be used from pkgsStatic
+  qemu-user = callPackage ../applications/virtualization/qemu/user.nix {};
```

Comment:
This should be inlined like the other variants, otherwise overriding won't work.

Reply:
Done. (Avoided spamming `all-packages.nix` with 30 lines of overrides by introducing a new option as you [previously suggested](pkgs/applications/virtualization/qemu/default.nix).)